person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id # getters for @id, @name, and @age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = generate_id # generate a unique ID for the person
```

Comment:
- [ OPTIONAL] I thibk it would be great if you create the id with a query like this instead of making a method also I can see that it is empty so you can do something like this instead ✅✨

` @id = Random.rand(1..1000)`